microsoft-edge/toc.yml

```
@@ -958,47 +958,48 @@
 # /DevTools
 
 # =============================================================================
-# if top-of-page title is longer, add it to displayName comma-delimited list of lookup keywords
     - name: Microsoft Edge extensions
       items:
         - name: Overview of Microsoft Edge extensions
           href: extensions-chromium/index.md
 
+# -------------------------------------
         - name: What's new
           items:
             - name: Released features for Microsoft Edge Add-ons
               href: extensions-chromium/whats-new/released-features.md
+                  # add blogs
 
             - name: Roadmap for Microsoft Edge Add-ons
               href: extensions-chromium/whats-new/roadmap.md
 
-        - name: Getting started
+# -------------------------------------
+        - name: Get started
           items:
-            - name: Extension concepts and architecture
+            - name: Extension concepts and architecture  # add "What is an extension?"
               href: extensions-chromium/getting-started/index.md
+#                   What are Edge Add-ons?  Include Manifest File and permissions
+
+#           - name: Extension type and usage
+#             href: 
 
             - name: Create an extension tutorial, part 1
               href: extensions-chromium/getting-started/part1-simple-extension.md
 
             - name: Create an extension tutorial, part 2
               href: extensions-chromium/getting-started/part2-content-scripts.md
 
+            - name: Port a Chrome extension to Microsoft Edge
+              href: extensions-chromium/developer-guide/port-chrome-extension.md
+
             - name: Sideload an extension
               href: extensions-chromium/getting-started/extension-sideloading.md
 
-        - name: Developer Guide
+# -------------------------------------
+        - name: Build an extension
```

```suggestion
        - name: Develop an extension
```
@sivMSFT @captainbrosset - a more standard word.  esp in APIs bucket we'll want conceptual sequenced distinction: 

APIs for Develop phase - i wouldn't call it Build phase.  'build' connotes narrow technical processing.
APIs for Publish phase